src/common/utils/modulesRegistry.h

```diff
@@ -68,13 +73,6 @@ inline registry::ChangeSet getMonacoPreviewHandlerChangeSet(const std::wstring i
                 for (uint32_t j = 0; j < extensionsList.Size(); ++j)
                 {
                     auto extension = extensionsList.GetStringAt(j);
-
```

Comment:
This is wrong. This will break the File explorer add-ons on/off logic.

Reply:
@stefansjfw Could you please provide a screenshot (or, if not applicable, other demonstration) of the broken logic? To fix something, I need to know what's broken first. Don't you agree?

Reply:
@skycommand, sorry for missing this.. The way you did it, it works.. My point of view was in case `languages.json` remains the same